docs/jade-getting-started.md

```diff
@@ -292,14 +322,36 @@ to build `jade-data-repo`.
 * Certain environment variables need to be set beforehand. Instances of `zzz`
 should be replaced by your initials or the environment (i.e. `dev`):
 
+
 ```
+export NVM_DIR="$HOME/.nvm"
+export JAVA_HOME=$(/usr/libexec/java_home)
+
 export VAULT_ADDR=https://clotho.broadinstitute.org:8200
-export PROXY_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
+export ENVIRONMENT=dev
 export GOOGLE_CLOUD_PROJECT=broad-jade-zzz
```

Comment:
```suggestion
export GOOGLE_CLOUD_PROJECT=broad-jade-zzz
```

We should either switch everything to `zz` or keep it as `zzz`

Reply:
Good point!